hbase-server/src/test/java/org/apache/hadoop/hbase/ipc/TestSimpleRpcScheduler.java

```diff
@@ -559,6 +561,25 @@ public void testCoDelScheduling() throws Exception {
     }
   }
 
+  @Test
+  public void testFastPathBalancedQueueRpcExecutorWithQueueLength0() throws Exception {
+    String name = "testFastPathBalancedQueueRpcExecutorWithQueueLength0";
```

Comment:
Use the `TestName` junit rule instead.

Reply:
@ndimiduk - thanks for the suggestion. I've replaced the constant with a TestName rule.